js/es6/interpreter.js

```diff
@@ -0,0 +1,94 @@
+import Assignment from './assignment';
+import { initFactory, operatorInstance, StopPlanOutException } from './ops/utils';
+import _ from "underscore";
+import $ from 'jquery';
+
+class Interpreter {
+	constructor(serialization, experiment_salt='global_salt', inputs={}, environment) {
+		this._serialization = serialization;
+		if (!environment) {
+			this._env = new Assignment(experiment_salt);
+		} else {
+			this._env = environment;
+		}
+		this.experiment_salt = this._experiment_salt = experiment_salt;
+		this._evaluated = false;
+		this._in_experiment = false;
+		this._inputs = _.clone(inputs);
+	}
+
+	in_experiment() {
+		return this._in_experiment;
+	}
+
+	set_env(new_env) {
+		this._env = $.extend(true, {}, new_env);
```

Comment:
this function doesn't appear to be used anywhere - might just remove it or implement some naive version of deep copy here instead of relying on jQuery for this. As of right now this is the only place where we use jQuery for anything.